man/kernel-install.xml

```diff
@@ -228,6 +228,18 @@
             <command>kernel-install</command> will use <literal>Default</literal> instead.</para>
           </listitem>
       </varlistentry>
+      <varlistentry>
+        <term>
+          <filename>/etc/machine-info</filename>
+        </term>
+          <listitem>
+            <para>If this file contains the <varname>KERNEL_INSTALL_MACHINE_ID</varname> variable,
+            <command>kernel-install</command> will use it as <replaceable>MACHINE-ID</replaceable> instead of
+            the contents of <filename>/etc/machine-id</filename>. If the variable is not found in
+            <filename>/etc/machine-info</filename>, <command>kernel-install</command> will try to save the
+            machine ID it uses to install to <varname>$BOOT</varname> to this file.</para>
+          </listitem>
+      </varlistentry>
```

Comment:
I think adding a note about the why/what problem this solves here (and maybe in the NEWS as well), would be nice, i.e. that this is mostly to make life easier for image builders.

Reply:
Yeah, I'll add a note in NEWS. We have some other changes that need documenting too.